orm/migrations.md

```diff
@@ -279,6 +280,14 @@ The `SoftDeletes` method adds a nullable `deleted_at` `TIMESTAMP` column. This c
 table.SoftDeletes()
 ```
 
+#### Custom column
+
+If you are using column types that framework does not support yet, you can use the `Custom` method to customize the field type:
+
+```go
+table.Custom("geometry", "geometry")
```

Comment:
https://github.com/goravel/framework/blob/0bb3350f528bee267900ba065f35c78cf1c73c6a/contracts/database/schema/blueprint.go#L19

```go
Column(column string, ttype string) ColumnDefinition
```
Should we rename the method Column to Custom?